src/app/components/about/about.component.scss

```diff
@@ -2,46 +2,80 @@
 @import './styles/mixins.scss';
 
 .about-container {
-  overflow-x: hidden;
-  text-align: center;
-  font-size:$fs-16;
-  font-weight:$fw-light;
-  img {
-    width: 90%;
-    min-height: 150px;
-  }
-  * {
-    color: $gray-darker;
-  }
+  padding: 40px;
+}
+
+.about-content-column {
+  margin-top: 68px;
+  box-sizing: border-box;
+  padding: 0 0.75rem;
+  min-height: 1px;
+  margin-left: auto;
+  left: 75px;
+  right: auto;
+}
+
+.about-title {
+  color: #4d4d4d;
+  margin: 1.14rem 0 0.912rem 0;
+  font-size: 34px;
+  font-weight: 500;
+}
+
+.about-content {
+  color: #4d4d4d;
+  font-size: 15px;
+  line-height: 1.5;
+  font-weight: normal;
+}
+
+.blue-text {
+  color: #2196F3 !important;
+}
+
+@include screen-small {
   .about-title {
-    font-size: $fs-24;
-    margin-top: 50px;
-    margin-bottom: 30px;
-    font-weight: $fw-semibold;
+    text-align: center;
   }
-  .row {
-    .about-content-column {
-      text-align: left;
-      padding-left: 100px;
-      padding-right: 100px;
-      .about-content-title {
-        font-size: $fs-18;
-        margin-bottom: 30px;
-        margin-top: 30px;
-        font-weight: $fw-semibold;
-      }
-      .about-content-body {
-        font-size: $fs-16;
-        line-height: 20px;
-        letter-spacing: 0.5px;
-      }
-    }
+  .about-content-column {
+    left: 0px;
   }
 }
 
-@include screen-small {
+@include screen-medium {
```

Comment:
Why it was required ?

Reply:
@Shekharrajak I added this to remove the horizontal scroll